### ISSUE TITLE
Match function does not work on cell values and constants

### ISSUE THREAD

**Issue by kemsky** [EPPlusSoftware/EPPlus#1152]
There is condition in function code that limits match to work only with ranges and there are tests only for ranges.

```
[Test]
public void Test()
{
    using (var excel = new ExcelPackage())
    {
        var worksheet = excel.AddSheet("Test Worksheet");
        var targetCell = worksheet.Cell(1,1);
        var validationCell = worksheet.Cell(1,2);

        validationCell.Formula = "ISNUMBER(MATCH(\"?*@?*.?*\", A1, 0))";

        targetCell.SetText("test@test.com");

        validationCell.Calculate();
        
        var result = validationCell.GetValue<bool>();
        
        Assert.AreEqual(true, result);
    }
}
```

**Comment by JanKallman** [comment]
This is a bug. We will provide a fix shortly.

**Comment by JanKallman** [comment]
Fixed in 6.2.12 and 7.0.1
